src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2041,12 +2040,14 @@ export class StudyViewPageStore {
             }
         }
 
+        let url = '/';
         if (!_.isEmpty(this.geneQueries)) {
             formOps.Action = 'Submit';
             formOps.gene_list = this.geneQueries.map(query => unparseOQLQueryLine(query)).join('\n');
+            url = '/results/legacy_submission';
         }
-
-        formSubmit('index.do', formOps, "_blank", 'post');
+        window.localStorage.setItem("legacyStudySubmission",JSON.stringify(formOps))
```

Comment:
could you do me a favor and wrap this in a function that we import and which abstracts away the actual submission method?  you could call it, submitToPage